Registration.jsx

```diff
@@ -0,0 +1,170 @@
+          config.headers["authorization"] = "Bearer " + data.accessToken;
+        }
+        return config;
+      },
+      (error) => {
+        return Promise.reject(error);
+      }
+    );
+      
+
+
+    const handleSubmit = async (e) => {
+      e.preventDefault();
+      try {
+        const res = await axios.post("/login", { username, password });
+        setUser(res.data);
+      } catch (err) {
+        console.log(err);
+      }
+    };
+
+    const handlePasswordChange = (e) => {
+        setPassword(e.target.value);
+    }
+    const handlePasswordConfirmationChange = (e) => {
+        setPasswordConfirmation(e.target.value);
+    }
+
+
+    const handleUsernameChange = (e) => {
+      setUsername(e.target.value);
+    }
+    const handleEmailChange = (e) => {
+        setEmail(e.target.value);
+      }
+
+    const submitLogin = () => {
+      axios.post('http://localhost:5000/api/user/login', {
+        email: username,
+        password: password,
+      }).then((response) => {
+        console.log(response);
+      });
+      history.push('/home/user')
+    };
+
+    useEffect(() => {
+        fetch('/api/aaaLogin.json')
+            .then(response => response.json())
+            .then(data => console.log(data));
+    }, []);
+  
+
+    return (
+      <div className="login">
+        <div className="wrapper">
+            <div className="input_wrapper">
+            <h2 className="title">Zebra Registration</h2>
+                <Card
+                    key="registration"
+                    raised={true}
+                    hoverable={true}
+                    size={Card.SIZE.COMPACT}>
+                    cardAlignment={Card.ALIGNMENT.CENTERED}
+                    <CardHeader content={
+                        {
+                            icon: Icon.TYPE.USER,
+                            title: 'Sign Up',
+                        }
+                    }/>
+
+                    <CardBody>
+                         <form onSubmit={handleSubmit}>
+                            <Input 
+                                value = {username}
+                                onChange={(e) => handleUsernameChange(e, 'user')}
+                                label={"Username"}
+                                required={true}
+                                autoComplete="false"
+                                type="username"
+                                className={"user"}
+                            />
+                            <Input 
+                                value = {email}
+                                onChange={(e) => handleEmailChange(e, 'email')}
+                                label={"Email"}
+                                required={true}
+                                autoComplete="false"
+                                type="email"
+                                className={"user"}
+                            />
+                            <Input
```

Comment:
what are the constraints on password field? We will need a help text or some other design to show password. This could go as a separate PR too